sdk/inkrecognizer/README.md

```diff
@@ -0,0 +1,50 @@
+# Azure InkRecognizer library for JavaScript
```

Comment:
Hi! This README doesn't seem to follow the guidelines! If this is expected, let me know.

Reply:
Fixed.